core/src/main/csharp/ch/cyberduck/core/CredentialManagerPasswordStore.cs

```diff
@@ -17,15 +17,14 @@
 //
 
 using ch.cyberduck.core;
-using ch.cyberduck.core.exception;
+using ch.cyberduck.core.preferences;
 using Ch.Cyberduck.Core.CredentialManager;
 using org.apache.logging.log4j;
 using System;
-using System.Collections.Generic;
-using System.Linq;
 using System.Net;
 using System.Text;
-using System.Threading.Tasks;
+using static Windows.Win32.Security.Credentials.CRED_PERSIST;
+using static Windows.Win32.Security.Credentials.CRED_TYPE;
 
 namespace Ch.Cyberduck.Core
 {
```

Comment:
Should probably just implement `HostPasswordStore` if overriding all implementations from `DefaultHostPasswordStore` anyways.

Reply:
I'd agree, unless there is logic in DefaultHostPasswordStore, that I don't want to replicate _again_. Thats why I overwrote the HostPasswordStore-implementation for the revised implementation, and falling back to DefaultHostPasswordStore-logic.
See original logic for CredentialManagerPasswordStore: https://github.com/iterate-ch/cyberduck/blob/master/core/src/main/csharp/ch/cyberduck/core/CredentialManagerPasswordStore.cs
I didn't want to have three different proxies in https://github.com/iterate-ch/cyberduck/blob/feature/GH-12805-1/core/src/main/csharp/ch/cyberduck/core/PasswordStoreFacade.cs.